cmd/root.go

```diff
@@ -29,7 +29,8 @@ func init() {
 	RootCmd.Flags().StringVarP(&opt.Password, "password", "p", "", "Optional: The password to connect to the openGemini cluster.")
 	RootCmd.Flags().StringVarP(&opt.DataDir, "from", "f", "/var/lib/influxdb/data", "Influxdb Data storage path. See your influxdb config item: data.dir")
 	RootCmd.Flags().StringVarP(&opt.Out, "to", "t", "127.0.0.1:8086", "Destination host to write data to")
-	RootCmd.Flags().StringVarP(&opt.Database, "database", "", "", "Optional: the database to read")
+	RootCmd.Flags().StringVarP(&opt.SrcDatabase, "src_database", "", "", "Optional: the database to read")
```

Comment:
```suggestion
	RootCmd.Flags().StringVarP(&opt.Database, "database", "", "", "Optional: the database to read")
```
没有必要改变参数，仅扩展`dst_database`参数即可。 默认 dst_database和原始db一致。

Reply:
因为 --retention 都没有改成src_retention，要改的话有点突兀，所以建议仅扩展语法。